src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/BlogPostItemPopulatorSpec.groovy

```diff
@@ -0,0 +1,41 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.moderation
+
+import org.apache.wicket.markup.repeater.Item
+import org.springframework.test.util.ReflectionTestUtils
+import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPost
+import pl.tomaszdziurko.jvm_bloggers.utils.NowProvider
+import spock.lang.Specification
+
+import java.time.LocalDateTime
+import java.time.Month
+
+/**
+ * @autor mszarlinski
+ */
```

Comment:
According to [general coding guidelines](https://github.com/tdziurko/jvm-bloggers/blob/master/CONTRIBUTING.md#general-coding-guidelines):

> @author tag shouldn't be placed in test classes